dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.ts

```diff
@@ -48,19 +61,31 @@ const mapAmendmentToLatestCompletedDate = (
   };
 };
 
-const mapAmendmentToLatestCompletedFacilityEndDate = (
-  amendment: TfmFacilityAmendment,
-): {
-  amendmentId: string;
-  facilityEndDate: string;
-} => {
-  const { amendmentId, facilityEndDate } = amendment;
-  if (!facilityEndDate) {
-    throw new Error('Found amendment does not have a defined facilityEndDate');
+const mapAmendmentToLatestCompletedFacilityEndDate = (amendment: TfmFacilityAmendment): CompletedFacilityEndDate => {
```

Comment:
I think having `latest` in this probably increases the confusion - It just takes one amendment and maps that right?

Reply:
Good point, have renamed